android/app/build.gradle

```diff
@@ -95,7 +95,7 @@ def enableSeparateBuildPerCPUArchitecture = false
 /**
  * Run Proguard to shrink the Java bytecode in release builds.
  */
-def enableProguardInReleaseBuilds = false
+def enableProguardInReleaseBuilds = true
```

Comment:
This should only be set to true for releases. For debugging (coding regularly prior to an update), it should be set to false.

By setting it to true in debugging mode, it may make it harder to use the error stacktrace, among other things.

Reply:
Make sure to eliminate other "shrink" rules as well. Overall, this makes methods names to be obfuscated and harder to read